docs/config/all.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Phinx documentation build configuration file, created by
+# sphinx-quickstart on Thu Jun 14 17:39:42 2012.
+#
+
+# Import the base theme configuration
+from cakephpsphinx.config.all import *
+
+# The full version, including alpha/beta/rc tags.
+release = '0.11.x'
```

Comment:
these might have to be fixed to 0.12 then

Reply:
Do you want me to change this or is this something that the cakephp folks will handle once 0.next is going to be merged into master?

Reply:
If you can would be nice, so we dont forget when merging

Reply:
Updated to 0.12.x